packages/packages/libs/editor-controls/src/controls/transition-control/__tests__/transition-repeater-control.test.tsx

```diff
@@ -74,10 +74,26 @@ describe( 'TransitionRepeaterControl', () => {
 		const props = { setValue, value, bind: 'transition', propType };
 
 		// Act
-		renderControl( <TransitionRepeaterControl recentlyUsedList={ [] } />, props );
+		renderControl( <TransitionRepeaterControl currentStyleState={ null } recentlyUsedList={ [] } />, props );
 
 		// Assert
 		const addButton = screen.getByLabelText( 'Add item' );
 		expect( addButton ).toBeInTheDocument();
 	} );
+
+	it( 'should display a disabled add button when not in normal style state', () => {
+		// Arrange
+		const setValue = jest.fn();
+		const value = { $$type: 'array', value: [] };
+		const propType = createTransitionPropType();
+		const props = { setValue, value, bind: 'transition', propType };
+
+		// Act
+		renderControl( <TransitionRepeaterControl currentStyleState={ 'hover' } recentlyUsedList={ [] } />, props );
+
+		// Assert
+		const addButton = screen.getByLabelText( 'Add item' );
+		expect( addButton ).toBeInTheDocument();
+		expect( addButton ).toBeDisabled();
+	} );
```

Comment:
Should we check the opposite scenario as well? E.g, `should display an active add button when in normal style state`
WDYT?

Reply:
Added enabled check to another test.
Fixed.